src/corehost/cli/deps_entry.cpp

```diff
@@ -104,8 +104,8 @@ bool deps_entry_t::to_full_path(const pal::string_t& base, pal::string_t* str) c
     }
 
     pal::string_t new_base = base;
```

Comment:
@joelverhagen, does the project.lock.json (transitively deps.json) preserve the casing or has it all small case?